server/src/test/java/org/elasticsearch/common/settings/SettingsModuleTests.java

```diff
@@ -83,6 +85,39 @@ public void testRegisterSettings() {
                 assertEquals("Failed to parse value [false] for setting [some.custom.setting]", ex.getMessage());
             }
         }
+        {
```

Comment:
Can we please add this as a new test method?

Reply:
Done, thanks.